man/__init__.py

```diff
@@ -33,5 +33,17 @@ def get_parser():
         "Oracle Linux 6/7/8, Scientific Linux 7, Alma Linux 8, and Rocky Linux 8 "
         "to the respective major version of RHEL.".strip()
     )
-    parser.usage = None
+    parser.usage = (
+        "  convert2rhel\n"
+        "  convert2rhel [-h]\n"
+        "  convert2rhel [--version]\n"
+        "  convert2rhel [-u username] [-p password | -c conf_file_path] [--pool pool_id | -a] [--disablerepo repoid]"
+        " [--enablerepo repoid] [--serverurl url] [--keep-rhsm] [--no-rpm-va] [--debug] [--restart]"
+        " [-y]\n"
+        "  convert2rhel [--no-rhsm] [--disablerepo repoid]"
+        " [--enablerepo repoid] [--no-rpm-va] [--debug] [--restart] [-y]\n"
+        "  convert2rhel [-k activation_key | -c conf_file_path] [-o organization] [--pool pool_id | -a] [--disablerepo repoid] [--enablerepo"
+        " repoid] [--serverurl url] [--keep-rhsm] [--no-rpm-va] [--debug] [--restart] [-y]\n"
+        r"  convert2rhel {analyze}"
+    )
```

Comment:
With my suggestion from here: https://github.com/oamg/convert2rhel/pull/820#issuecomment-1686462026 we don't need to change the usage anymore before generating the manpage. It will get formatted properly without any interference from us. 

```suggestion
```